lib/autoupdate/agent/installer.go

```diff
@@ -89,6 +88,8 @@ type LocalInstaller struct {
 	// ValidateBinary returns true if a file is a linkable binary, or
 	// false if a file should not be linked.
 	ValidateBinary func(ctx context.Context, path string) (bool, error)
+	// template is download URI template of Teleport packages.
+	template string
```

Comment:
```suggestion
	Template string
```
Export this -- it's an input like the other properties